package.json

```diff
@@ -44,7 +44,8 @@
     "changelog": "conventional-changelog --preset angular --infile CHANGELOG.md --same-file",
     "changelog:unreleased": "conventional-changelog --preset angular --output-unreleased",
     "release:branch": "scripts/release-branch.sh",
-    "release:publish": "scripts/release-publish.sh"
+    "release:publish": "scripts/release-publish.sh",
```

Comment:
both these scripts should be removed from package.json & the file system :)

Reply:
Yes, but I was thinking of removing them in a separate PR once you use Ship.js in this repo and it works as you expect. I can remove them here if you want :)

Reply:
If shipjs doesn't work we can always revert this commit, otherwise we will forget to remove it I think

Reply:
okay good point